conf/app.routes

```diff
@@ -94,6 +94,9 @@ GET         /:taxYear/annual-allowance/pension-scheme-tax-reference-summary
 GET         /:taxYear/pension-income/state-pension-lump-sum-tax                             controllers.pensions.incomeFromPensions.TaxPaidOnStatePensionLumpSumController.show(taxYear: Int)
 POST        /:taxYear/pension-income/state-pension-lump-sum-tax                             controllers.pensions.incomeFromPensions.TaxPaidOnStatePensionLumpSumController.submit(taxYear: Int)
 
+GET         /:taxYear/pension-income/state-pension-lump-sum-tax-amount                      controllers.pensions.incomeFromPensions.TaxPaidOnLumpSumAmountController.show(taxYear: Int)
+POST        /:taxYear/pension-income/state-pension-lump-sum-tax-amount                      controllers.pensions.incomeFromPensions.TaxPaidOnLumpSumAmountController.submit(taxYear: Int)
+
```

Comment:
I found these income from pension ones really difficult to tell apart

Could we order them as per page flow to help?
i.e. as below
GET         /:taxYear/pension-income/state-pension                      
POST        /:taxYear/pension-income/state-pension                      
GET         /:taxYear/pension-income/state-pension-amount               
POST        /:taxYear/pension-income/state-pension-amount               
GET         /:taxYear/pension-income/state-pension-lump-sum             
POST        /:taxYear/pension-income/state-pension-lump-sum             
GET         /:taxYear/pension-income/state-pension-lump-sum-amount      
POST        /:taxYear/pension-income/state-pension-lump-sum-amount      
GET         /:taxYear/pension-income/state-pension-lump-sum-tax         
POST        /:taxYear/pension-income/state-pension-lump-sum-tax         
GET         /:taxYear/pension-income/state-pension-lump-sum-tax-amount  
POST        /:taxYear/pension-income/state-pension-lump-sum-tax-amount